lib/scope.js

```diff
@@ -1508,7 +1531,8 @@ module.exports = {
           let source_row = row;
           if ( row.source ) { source_row = row.source; }
           let original_row = source_row.original;
-          let msg = `Did not find a field on this page for the variable \`${ original_row.var }\` that could be set to \`${ original_row.value }\``;
+          let reported_val = printable_var_value(original_row); 
+          let msg = `Did not find a field on this page for the variable \`${ original_row.var }\` that could be set to \`${ reported_val }\``;
```

Comment:
From what I can tell, this won't result in what you'd hope. E.g. for a row like this:
```
| some_var | SOME_SECRET |  |
```
Testing it out in the console, it resulted in "Did not find a field on this page for the variable `some_var` that could be set to `${SOME_SECRET}`". I'm also taking the opportunity to get rid of my own past inconsistency with those backticks unless someone objects.

```suggestion
          let msg = `Did not find a field on this page for the variable "${ original_row.var }" that could be set to "${ original_row.value }"`;
```

Reply:
The same situation shows up elsewhere and I can grab those too if we determine that I'm not missing something.

Reply:
I don't think it matters too much. The backticks are shown in console, and IMO backtics represent any kind of verbatim code. But I'll take the suggestions anyway.